frontend/src/todos/todoListPresenter.tsx

```diff
@@ -0,0 +1,32 @@
+import TodoPresenter from './todoPresenter';
+import { Todo } from './todo';
+
+export interface TodosListProps {
+  todos: Todo[];
+  isFinished: boolean;
+  onToggle: (id: string) => void;
+  onDelete: (id: string) => void;
+}
+
+function TodosListPresenter(props: TodosListProps) {
+  const onToggle = props.onToggle;
+  const onDelete = props.onDelete;
+  return (
+    <div className={props.isFinished ? 'finished' : 'notFinished'}>
+      <h2>{props.isFinished ? 'Finished Tasks' : 'Not Finished Tasks'}</h2>
+      <ul>
+        {props.todos
+          .filter((todo) => todo.isFinished === props.isFinished)
+          .map((todo) => (
+            <TodoPresenter
+              todo={todo}
+              onToggle={() => onToggle(todo.id)}
+              onDelete={() => onDelete(todo.id)}
+            />
```

Comment:
when rendering lists using map the component must receive a "key" prop, this helps react render (if you will open the dev tools you will see that it complains)